app/models/work.rb

```diff
@@ -0,0 +1,17 @@
+class Work < ApplicationRecord
+
+  has_many :votes, dependent: :destroy
+  has_many :users, through: :votes
+
+  validates :title, presence: true, uniqueness: {scope: :category}
+  validates :description, presence: true
+  validates :publication_date, presence: true, numericality: true
+  validates :creator, presence: true
+  validates :category, presence: true, inclusion: { in: %w(album book movie), message: "category must be a movie, book or album" }
+
+
+  def self.order_media(category)
+    where(category: category).order(vote_count: :desc)
+  end
```

Comment:
This works, but it would be easy for the `vote_count` attribute and the number of votes in the system to get out of alignment.  For example someone in the DB deletes a vote, now they don't match!

It's better to rely on `votes.count` and sort using [the relationship with a bit of research.](https://stackoverflow.com/questions/16996618/rails-order-by-results-count-of-has-many-association)